src/App.css

```diff
@@ -68,3 +69,27 @@ input {
 .card {
   display: inline-block;
 }
+.result_title{
+   font-size: 32px;
+   font-family: 'Open Sans', sans-serif;
```

Comment:
The project has already 4 different fonts. This one is nice, but I don't think it should be defined on the line level. What do you think? maybe redefine for the whole project

Reply:
cool suggestion... I was looking at open sans for the h3 title and a more jolly one like poppins for the body (Line level)... Ill make them global like you said. thanks